From 7e8585ce969b760f5f0b9395b69ea6ea315036a2 Mon Sep 17 00:00:00 2001
From: Gary Ching-Pang Lin <chingpang@gmail.com>
Date: Sat, 24 Oct 2015 14:10:16 +0800
Subject: [PATCH] use the right qt5 iid

---
 qt5-im/gcin-qt5.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qt5-im/gcin-qt5.h b/qt5-im/gcin-qt5.h
index 63aacc0..0f958af 100644
--- a/qt5-im/gcin-qt5.h
+++ b/qt5-im/gcin-qt5.h
@@ -9,7 +9,7 @@ class QGcinPlatformInputContextPlugin : public QPlatformInputContextPlugin
 {
     Q_OBJECT
 public:
-    Q_PLUGIN_METADATA(IID "org.qt-project.Qt.QPlatformInputContextFactoryInterface" FILE "gcin.json")
+    Q_PLUGIN_METADATA(IID QPlatformInputContextFactoryInterface_iid FILE "gcin.json")
     QStringList keys() const;
     QGcinPlatformInputContext *create(const QString& system, const QStringList& paramList);
 };
-- 
2.6.2

