diff -Naur gcin-2.7.8-orig/configure gcin-2.7.8/configure
--- gcin-2.7.8-orig/configure	2012-06-24 16:21:54.000000000 +0200
+++ gcin-2.7.8/configure	2012-09-02 11:57:22.590572280 +0200
@@ -174,11 +174,11 @@
 
 
 LIB='lib'
-if [ -d /usr/lib64 -a -d /lib64 -a ! -d /lib/x86_64-linux-gnu ]; then
-LIB='lib64'
+#if [ -d /usr/lib64 -a -d /lib64 -a ! -d /lib/x86_64-linux-gnu ]; then
+#LIB='lib64'
 # for fedora 64bit
-GTK_QUERY_IM_64='-64'
-fi
+#GTK_QUERY_IM_64='-64'
+#fi
 
 QT_IM='N'
 QT_IM_DIR=/usr/$LIB/qt3/plugins/inputmethods
